augly/video/functional.py

```diff
@@ -2137,3 +2139,58 @@ def vstack(
         helpers.get_metadata(metadata=metadata, function_name="vstack", **func_kwargs)
 
     return output_path or video_path
+
+
+def augment_audio(
+    video_path: str,
+    audio_aug_function: Callable[..., Tuple[np.ndarray, int]] = audaugs.apply_lambda,
+    output_path: Optional[str] = None,
+    metadata: Optional[List[Dict[str, Any]]] = None,
+) -> str:
+    """
+    Augments audio into a video
+
+    @param video_path: the path to the video to be augmented
+
+    @param audio_aug_function: the augmentation function to be applied onto the video
+        (should expect a video path and output path as input and output the augmented
+        video to the output path. Nothing needs to be returned) THIS NEEDS TO BE CHANGED
+
+    @param output_path: the path in which the resulting video will be stored.
+        If not passed in, the original video file will be overwritten
+
+    @param metadata: if set to be a list, metadata about the function execution
+        including its name, the source & dest duration, fps, etc. will be appended
+        to the inputted list. If set to None, no metadata will be appended or returned
+
+    @returns: the path to the augmented video
+    """
+    assert callable(audio_aug_function), (
+        repr(type(audio_aug_function).__name__) + " object is not callable"
+    )
+
+    func_kwargs = helpers.get_func_kwargs(metadata, locals(), video_path)
+
+    audio_metadata: Optional[List[Dict[str, Any]]] = None
```

Comment:
We want to pass this into the `audio_aug` as `[]`, **not** `None`, which would mean that nothing would be appended to it.
```suggestion
    audio_metadata = []
```